src/bulk/common.ts

```diff
@@ -923,6 +923,32 @@ export interface BulkWriteOptions extends CommandOperationOptions {
   forceServerObjectId?: boolean;
 }
 
+/**
+ * TODO(NODE-XXX)
```

Comment:
marking this to make sure we don't leave a placeholder TODO

Reply:
done!